wasm/README.md

```diff
@@ -46,13 +39,43 @@ WebSocket implementation. Two of such modules are [WebSocket](https://www.npmjs.
 (provides a custom implementation) and [isomorphic-ws](https://www.npmjs.com/package/isomorphic-ws) 
 (built on top of the ws WebSocket module).
 
-You can use the following shims:
 
-```js
-// `websocket` module
+## Loading in a Web App
+
+```html
+<html>
+    <head>
+        <script type="module">
+            import * as kaspa_wasm from './kaspa/kaspa-wasm.js';
+            (async () => {
+                const kaspa = await kaspa_wasm.default('./kaspa/kaspa-wasm_bg.wasm');
+            })();
+        </script>
+    </head>
+    <body></body>
+</html>
+```
+
+## Loading in a Node.js App
+
+```javascript
+// W3C WebSocket module shim
 globalThis.WebSocket = require('websocket').w3cwebsocket;
-// `isomorphic-ws` module
-globalThis.WebSocket = require('isomorphic-ws');
+
+let {RpcClient,Encoding,init_console_panic_hook,defer} = require('./kaspa-rpc');
+// init_console_panic_hook();
+
+let rpc = new RpcClient(Encoding.Borsh,"ws://127.0.0.1:17110");
+
+(async () => {
+    await rpc.connect();
+
+    let info = await rpc.getInfo();
+    console.log(info);
+
+    await rpc.disconnect();
```

Comment:
Optional, just to follow some good practices even in examples.

```suggestion
    try {
        let info = await rpc.getInfo();
        console.log(info);
    } finally {
        await rpc.disconnect();
    }
```

Reply:
Added try/finally